crates/forge_loader/src/manifest.rs

```diff
@@ -416,4 +529,159 @@ mod tests {
             }
         );
     }
+
+    // Modified specific deserialization schemes for modules. Checking that new schemes can deserialize function values.
+    #[test]
+    fn test_new_deserialize() {
+        let json = r#"{
+            "app": {
+                "name": "My App",
+                "id": "my-app"
+            },
+            "modules": {
+                "macro": [
+                {
+                    "key": "my-macro",
+                    "title": "My Macro",
+                    "function": "Catch-me-if-you-can0", 
+                    "resolver": "Catch-me-if-you-can1",
```

Comment:
<!--
 Copyright 2024 Joshua Wong.
 SPDX-License-Identifier: Apache-2.0
-->
See <https://github.com/atlassian-labs/FSRT/pull/31#discussion_r1451932098> for how the `resolver` property should look. Also, can you add test cases for when there's a `resolver`, but no outer `function` to make sure we handle the case I mentioned in <https://github.com/atlassian-labs/FSRT/pull/31#discussion_r1451932098>?

```suggestion
                    "resolver": {
                        "function": "Catch-me-if-you-can1"
                    },
```